set.mm

```diff
@@ -17780,6 +17780,16 @@ disjoint variable condition ( ~ sb2 ).  Theorem ~ sb6f replaces the
       ( weq wi wsb sb2vOLD id mpg ) ABCZIDIABEAIABFIGH $.
   $}
 
+  ${
+    $d x t $.
+    $( One implication of ~ sbn with a disjoint variable condition on
+       ` x , t ` , requiring fewer axioms.  (Contributed by NM, 14-May-1993.)
+       Adapted from ~ sbn .  (Revised by BJ, 18-Sep-2024.) $)
+    sbnv $p |- ( [ t / x ] -. ph -> -. [ t / x ] ph ) $=
+      ( wn wsb weq wa wex wi wal sb6 alinexa bitri sb1v con3i sylbi ) ADZBCEZBC
+      FZAGBHZDZABCEZDRSQIBJUAQBCKSABLMUBTABCNOP $.
+  $}
+
```

Comment:
did you see [sbn1](https://us.metamath.org/mpeuni/sbn1.html)?

Reply:
Obviously, I haven't. It looks like sbn1 actually subsumes sbnv, so I'm going to use your version.

Reply:
(I use the "rocket" emoji for "done".)

I replaced nsb with its closed form. I hope it's ok with you.